docker/Dockerfile.ubuntu.xenial

```diff
@@ -50,14 +45,17 @@ RUN apt-get update && apt-get -y install --no-install-suggests --no-install-reco
   opensc \
   pkg-config \
   psmisc \
+  python-dev \
```

Comment:
It's pretty annoying that we need python2 and python3. What's python2 required for?

Reply:
Python2 is required to build boost but in the final PR I will change it to be possible to build boost with Python3

Reply:
Is this still part of the plan?

Reply:
No, this needed only for building boost with enabled boost::python module, we do not use it right now, so it can be removed